### PR TITLE
S2930: Add FN repro for #7635

### PR DESCRIPTION
**Pull request by martin-strecker-sonarsource**
Reproducer for #7635

### CODE REVIEW

**Comment by pavel-mikula-sonarsource** [comment]
I'd name the PR `S2930: Add FN repro for #7635` to have consistent commit history.

**Review by antonioaversa — CHANGES_REQUESTED** [review]
Some more testing to capture the generic interface

File: analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DisposableNotDisposed.ILogger.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
```
[the remark this answers is not in the document]
Comment (martin-strecker-sonarsource) [reply analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DisposableNotDisposed.ILogger.cs:2]: Now it is ;:)

**Review by antonioaversa — APPROVED** [review]
LGTM

**Comment by antonioaversa** [comment]
@martin-strecker-sonarsource : Master is currently frozen. I let you merge this one, once the release is done and the branch is unfrozen.

**Pull request merged**